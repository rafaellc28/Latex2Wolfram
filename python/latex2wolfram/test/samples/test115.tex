% Test LU decomposition
LU(\begin{bmatrix}
   7 & 3 & -11\\
  -6 & 7 &  10\\
 -11 & 2 & -2\\
\end{bmatrix})
