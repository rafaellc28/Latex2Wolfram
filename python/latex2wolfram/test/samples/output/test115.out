LU decomposition of({{7, 3, -11}, {-6, 7, 10}, {-11, 2, -2}})
